### PR TITLE
chore: add docker config

### PR DESCRIPTION
**Pull request by TomWoodward**
@philschatz i moved your dev server to a new home so i could share it with the docker config. in the future we'll add proxy middleware that can be shared between docker, tests, and dev.

also jq was just straight not working so i replaced with a custom postinstall script

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM node:10.9-alpine
+
+COPY . /code
+WORKDIR /code
+RUN yarn && yarn build
+
+EXPOSE 8000
+
+CMD yarn server
```
Comment (philschatz) [inline Dockerfile]:
```suggestion
CMD yarn run server
```

File: Dockerfile [hunk 2]
```diff
@@ -0,0 +1,9 @@
+FROM node:10.9-alpine
+
+COPY . /code
+WORKDIR /code
+RUN yarn && yarn build
```
Comment (philschatz) [inline Dockerfile]: Just to clarify to people not necessarily familiar with yarn's commandline syntax. What do you think?

```suggestion
RUN yarn install
RUN yarn run build
```

**Review by philschatz — CHANGES_REQUESTED** [review]
Yay! It works for me. Should docker instructions be included in the README?

**Comment by TomWoodward** [comment]
@philschatz added readme and rejiggered the structure a little bit.

i made `yarn run` the entrypoint so that in the future we could do `docker run openstax/books-web pre-render`. `server` is still the default command.

i almost added `docker run openstax/books-web test` as an example command but it doesn't work. i'll need to extend the dockerfile a bit to install chrome and stuff for puppeteer to work. i'll work on that tomorrow.

**Comment by TomWoodward** [comment]
ui tests work in the docker container now

**Review by philschatz — APPROVED** [review]

**Pull request merged**